cli/crates/server/src/errors.rs

```diff
@@ -16,9 +16,11 @@ use crate::udf_builder::JavaScriptPackageManager;
 
 #[derive(Error, Debug)]
 pub enum JavascriptPackageManagerComamndError {
+    #[error("working directory '{0}' not found")]
+    WorkingDirectoryNotFound(PathBuf),
     /// returned if npm/pnpm/yarn cannot be found
     #[error("could not find {0}: {1}")]
-    NotFound(JavaScriptPackageManager, which::Error),
+    NotFound(JavaScriptPackageManager, String),
```

Comment:
Why did we change this to a string? It's normally better to keep the original error to be able to display it or display hints by matching it

Reply:
I've changed it back.
The original reason was that I was reusing the variant `NotFound` for case when working directory is not found. 
However if this was a public library I would advise against leaking the dependency for better backwards compatibility.